core/dovecot/conf/dovecot.conf

```diff
@@ -6,6 +6,11 @@ protocols = imap pop3 lmtp sieve
 postmaster_address = {{ POSTMASTER }}@{{ DOMAIN }}
 hostname = {{ HOSTNAMES.split(",")[0] }}
 submission_host = {{ FRONT_ADDRESS }}
+{%- if SUBNET6 -%}
+listen = *,::
```

Comment:
Is "*,::" necessary? Would it be sufficient to specify "::"?

Reply:
I think it is. We mirror dovecot's defaults here: https://doc.dovecot.org/settings/core/#core_setting-listen